src/Main.tsx

```diff
@@ -93,6 +94,29 @@ const App: React.FunctionComponent = () => {
   const config = useConfig();
   const explorer = useKnockoutExplorer(config?.platform, explorerParams);
 
+  const splitterBounds: SplitterBounds = {
+    min: ExplorerMetrics.SplitterMinWidth,
+    max: ExplorerMetrics.SplitterMaxWidth,
+  };
+
+  const splitter = new Splitter({
+    splitterId: "h_splitter1",
+    leftId: "resourcetree",
+    bounds: splitterBounds,
+    direction: SplitterDirection.Vertical,
+  });
```

Comment:
I just tried this in production and the splitter does not work even before this change. Remove it entirely.

Reply:
Removed